.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,40 @@
-# Description of the problem
+# Description of the issue
```

Comment:
I am unsure if the word issue is the right thing here right. Because sometimes a pull request does not fix an issue.
Maybe the phrasing "Description of the content of the pull request"

It could be possible to merge the two paragraph.
Like description of the content and then we explain if this is a fix what is the issue and how it is fixed. If it's a feature explain what is the feature and what it accomplishes

Reply:
I kinda like the distinction between problem/solution. When I reread the commit message while debugging it is easier to target what was done since I don't really need to know why it was done. I really hate when people write "this is fixed" in the solution tho :P 
I agree with "content of PR"

Reply:
Yup agreed. Maybe this is something I can fix though. I can remind people that  PR messages should be more descriptive. I am also guilty of this to be honest. I will be more careful from now on.